src/core/service.c

```diff
@@ -929,8 +929,11 @@ static int service_load_pid_file(Service *s, bool may_warn) {
         prio = may_warn ? LOG_INFO : LOG_DEBUG;
 
         fd = chase_symlinks(s->pid_file, NULL, CHASE_OPEN|CHASE_SAFE, NULL);
-        if (fd == -EPERM)
-                return log_unit_full(UNIT(s), prio, fd, "Permission denied while opening PID file or unsafe symlink chain: %s", s->pid_file);
+        if (fd == -EPERM) {
+                log_unit_full(UNIT(s), prio, fd, "Permission denied while opening PID file or unsafe symlink chain. Fallback to unsafe logic: %s", s->pid_file);
```

Comment:
we should probably have this is recognizable error that carries an explanation in the message catalog. i.e. use log_struct(…,  "MESSAGE_ID=" …